src/lib/configwidgetslib/erroroverlay.cpp

```diff
@@ -41,6 +44,11 @@ void ErrorOverlay::availabilityChanged(bool avail) {
     }
 }
 
+void ErrorOverlay::runFcitx5()
+{
+    system("fcitx5-autostart");
```

Comment:
1. Just run fcitx5.
2. don't use system().
See how FcitxModule::runFcitx() implement it.